### DISCUSSION TITLE
Is PURGING a default feature?

### DISCUSSION THREAD

**Discussion by ilyasseisov** [tailwindlabs/tailwindcss#13990] [Help] [answered]
Hi friends👋
I'm a newbie at Tailwind CSS. 
Could you please clarify for me the following:

I'm using Tailwind CSS in my **React project**.
After I build the project, i get the final CSS file.

Does this file have only classes I use?
or
I must manually purge them?

Thanks in advance!

**Comment by RobinMalfait** [comment, marked as answer]
Hey!

Tailwind CSS doesn't have a config of purging in Tailwind CSS v3 and up, however, it works by only generating CSS for classes found in the files you listed in your `content` array (https://tailwindcss.com/docs/content-configuration) which means that you don't need to do purging yourself 👍

**Comment by ilyasseisov** [comment]
Thank you very much! 🙂

<https://ilyasseisov.com/>

Ilyas Seisov

Front-End Web Developer
[image: globe]

ilyasseisov.com
[image: linkedin] <https://linkedin.com/in/ilyasseisov1> [image: twitter]
<https://twitter.com/ilyasseisov1>
